functests/ptp/ptp.go

```diff
@@ -144,6 +154,61 @@ var _ = Describe("ptp", func() {
 	})
 })
 
+// Returns the slave node label to be used in the test, empty string label cound not be found
+func checkPolicies(namespace string) string {
+	var masters []ptpv1.PtpConfig
+	var slaves []ptpv1.PtpConfig
+
+	configList, err := client.Client.PtpConfigs(namespace).List(metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for _, config := range configList.Items {
+		for _, profile := range config.Spec.Profile {
+			if isPtpMaster(*profile.Ptp4lOpts, *profile.Phc2sysOpts) {
+				masters = append(masters, config)
+			}
+			if isPtpSlave(*profile.Ptp4lOpts, *profile.Phc2sysOpts) {
+				slaves = append(masters, config)
+			}
+		}
+	}
+	if len(masters) == 0 {
```

Comment:
Still open discussion: do we need masters?
@SchSeba ^^^^

Reply:
We should support both modes.
1. if there is a master policy in the cluster we validate that all the slaves are able to sync from it (not from any other server)
2. if no master policy we check all the slaves are able to sync from the same master

Reply:
For being able to run in discovery mode we only need slaves.

What we want is to make sure that, in discovery mode, if a master is running on the cluster then the slaves are synced to it and not to some other server. 
This is covered by this test https://github.com/openshift/ptp-operator/blob/a8f94f9591dba78c4bd7fe7a2cc8feb5cb55927e/test/ptp/ptp.go#L228 

So, let's remove masters from discovery mode and make sure that test works when a master is discovered on the cluster.
@mmirecki @SchSeba sounds good?

Reply:
+1